builtin/common/misc_helpers.lua

```diff
@@ -735,4 +719,4 @@ function core.privs_to_string(privs, delim)
 end
 
 assert(core.string_to_privs("a,b").b == true)
-assert(core.privs_to_string({a=true,b=true}) == "a,b")
+assert(core.privs_to_string({a=true,b=true}) == "a,b")
```

Comment:
Not high priority, but please configure your IDE to add a newline at the end of the file.